bin/configs/kotlin-spring-cloud.yaml

```diff
@@ -0,0 +1,12 @@
+generatorName: kotlin-spring
+outputDir: samples/client/petstore/kotlin-spring-cloud
```

Comment:
Can you please add this new folder to .github/workflows/samples-kotlin-client.yaml so that the CI will test it moving forward?

Reply:
added :)